config/locales/defaults/date.sv.yml

```diff
@@ -102,3 +102,12 @@ sv:
       long: "%Y/%m %d %H:%M:%S"
       short: "%d %b %H:%M"
     pm: ''
+  date_range:
+    long:
+      different_months_same_year: "%{from_day} %{from_month} %{sep} %{until_day} %{until_month}, %{year}"
+      different_years: "%{from_day} %{from_month}, %{from_year} %{sep} %{until_day} %{until_month}, %{until_year}"
+      same_month: "%{from_day} %{sep} %{until_day} %{month}, %{year}"
+    short:
+      different_months_same_year: "%{from_month} %{from_day} %{sep} %{until_month} %{until_day}, %{year}"
+      different_years: "%{from_month} %{from_day}, %{from_year} %{sep} %{until_month} %{until_day}, %{until_year}"
+      same_month: "%{from_day} %{sep} %{until_day} %{month}, %{year}"
```

Comment:
Will these strings be different in different locales? If not, no need to have them defined again for every language.